src/app/shell/admin-tools/data/directions-wrapper/directions-wrapper.component.html

```diff
@@ -3,13 +3,26 @@
     <mat-tab label="Напрямки для користувача">
       <app-directions></app-directions>
     </mat-tab>
-    <mat-tab label="МОН">
+    <mat-tab label="{{mainInstitutionTitles.MON}}">
+      <ng-template matTabContent>
+        <app-directions-institution-hierarchies-list [institutionTitle]="mainInstitutionTitles.MON"
+          [displayedColumns]="mainInstitutionDisplayColumns.MONDisplayColumns">
+        </app-directions-institution-hierarchies-list>
+      </ng-template>
     </mat-tab>
-    <mat-tab label="МКІП">
+    <mat-tab label="{{mainInstitutionTitles.MKIP}}">
```

Comment:
MKIP --> MKPI

and as I mentioned above, I expected to create this table dynamically, cause titles and amount of levels can be changed (this functionality will be implemented later)